### ISSUE TITLE
error: Cannot read property 'removeChild' of null when there is no other user in a session

### ISSUE THREAD

**Issue by vivevio** [OpenVidu/openvidu#129]
When in a session just left 1 user, I got the error in console,
`error: Cannot read property 'removeChild'`

Here the details:
<img width="1440" alt="screen shot 2018-10-12 at 2 09 10 pm" src="https://user-images.githubusercontent.com/7406182/46853954-c04c3180-ce29-11e8-8cde-ac057e76df88.png">

**Comment by pabloFuente** [comment]
May I see your JavasCript code? Especially where you initialize your Publisher object. That error is probably taking place when OpenVidu Browser tries to remove its associated video element from the DOM.

**Comment by vivevio** [comment]
This is my HTML:
`<div id="callVideoPlaceholder" class="br-video-container"></div>`

And this is my Javascript code:
```
let $videoPlaceholder   = $('#callVideoPlaceholder');

$videoPlaceholder.html(`
                    <div id="spotlight" class="br-spotlight"><video autoplay></div>
                    <div id="local-cam" class="br-local-cam"></div>
`);

OV_SESSION.connect(token)
                .then(() => {
                    
                    OV_PUBLISHER = OV.initPublisher("local-cam");

                    OV_PUBLISHER.on('videoElementCreated', function (event) {
                        event.element['muted'] = true;      
                    });
                    
                    OV_SESSION.publish(OV_PUBLISHER);

                })
                .catch(error => {
                    console.log("There was an error connecting to the session:", error.code, error.message);
                });
```

**Comment by vivevio** [comment]
1 on 1 call and also 1 to many. With the condition, user leaves the session while the other still connected.

**Comment by pabloFuente** [comment]
How are you subscribing to remote videos? I mean, the part of your code where:

```javascript
session.on('streamCreated', event => {
  session.subscribe(event.stream, 'subscriber');
});
```

Maybe there's something strange happening with jQuery. Can I see a screenshot or a copy-paste of the HTML code once the 1to1 call is running? I need to see the structure of the subscriber videos in the DOM.

To give you more info, the error is happening in [this exact line](https://github.com/OpenVidu/openvidu/blob/master/openvidu-browser/src/OpenVidu/StreamManager.ts#L363): `video.parentNode!.removeChild(streamManagerVideo.video);`

When you receive `streamDestroyed` event for one Subscriber (the user leaving the session triggers this event in the rest of users subscribed to its stream), that line tries to automatically remove from DOM all the videos displaying that Subscriber stream. The association is done in the call to `Session.subscribe` or `Subscriber.addVideoElement` or `Subscriber.createVideoElement`. All those associated videos will be removed from DOM by that line when receiving `streamDestroyed` event.

**Comment by L-BuenoSRP** [comment]
Also I have the same problem, I made my code based on the openvidu-js-node example. when the connection starts, after a few seconds it drops with the same message.
Here is the browser console log.

`'OpenVidu' initialized
Connection.ts:88 'Connection' created (local)
Connection.ts:141 Remote 'Connection' with 'connectionId' [zukmhcm2w2ymmcyj] is now configured for receiving Streams with options:  {id: "zukmhcm2w2ymmcyj_CAMERA_CUAOJ", createdAt: 1561143500296, connection: Connection, hasAudio: true, hasVideo: true, …}
Connection.ts:88 'Connection' created (remote) with 'connectionId' [zukmhcm2w2ymmcyj]
Session.ts:565 Event 'streamCreated' triggered by 'Session' StreamEvent {hasBeenPrevented: false, cancelable: false, target: Session, type: "streamCreated", stream: Stream, …}
200:817 event streamCreated [object Object]
Session.ts:242 Subscribing to zukmhcm2w2ymmcyj
StreamManager.ts:146 Event 'videoElementCreated' triggered by 'Subscriber' VideoElementEvent {hasBeenPrevented: false, cancelable: false, target: Subscriber, type: "videoElementCreated", element: video#remote-video-zukmhcm2w2ymmcyj_CAMERA_CUAOJ}
200:822 event videoElementCreated2 [object Object]
Session.ts:246 Subscribed correctly to zukmhcm2w2ymmcyj
StreamManager.ts:146 Event 'videoElementCreated' triggered by 'Publisher' VideoElementEvent {hasBeenPrevented: false, cancelable: false, target: Publisher, type: "videoElementCreated", element: video#local-video-undefined}
200:850 event videoElementCreated [object Object]
StreamManager.ts:126 Your local 'Stream' with id [undefined] video is now playing
Stream.ts:734 'Publisher' successfully published to session
Session.ts:565 Event 'streamDestroyed' triggered by 'Session' StreamEvent {hasBeenPrevented: false, cancelable: true, target: Session, type: "streamDestroyed", stream: Stream, …}
200:828 event streamDestroyed [object Object]
StreamEvent.ts:69 Calling default behavior upon 'streamDestroyed' event dispatched by 'Session'
Stream.ts:468 Inbound WebRTCPeer from 'Stream' with id [zukmhcm2w2ymmcyj_CAMERA_CUAOJ] is now closed
Stream.ts:484 Remote MediaStream from 'Stream' with id [zukmhcm2w2ymmcyj_CAMERA_CUAOJ] is now disposed
Session.ts:686 TypeError: Cannot read property 'removeChild' of null
    at StreamManager.ts:386
    at Array.forEach (<anonymous>)
    at Subscriber.StreamManager.removeAllVideos (StreamManager.ts:380)
    at StreamEvent.callDefaultBehavior (StreamEvent.ts:92)
    at Session.ts:674`
